de.metas.swat/de.metas.swat.base/src/test/java/de/metas/adempiere/service/impl/AddressBuilderTest.java

```diff
@@ -56,6 +58,8 @@ public static void staticInit()
 	public void init()
 	{
 		AdempiereTestHelper.get().init();
+		
+		org = prepareAD_Org();
```

Comment:
but now we don't have any test for the case when org is null (regression). shouldn't we have?